components/turret.py

```diff
@@ -154,9 +174,14 @@ def has_index(self) -> bool:
         return self.index_found
 
     def execute(self) -> None:
-        self._handle_indices()
+        # self._handle_indices()
```

Comment:
Is this intentionally commented out?

Reply:
Ditto. This can't be right. Indices won't be found without it. Or are we missing something?